src/Source/DoctrineORMQuerySourceIterator.php

```diff
@@ -155,7 +155,7 @@ protected function getValue($value)
     {
         if (is_array($value) || $value instanceof \Traversable) {
             $value = null;
-        } elseif ($value instanceof \DateTime) {
+        } elseif ($value instanceof \DateTimeInterface) {
```

Comment:
This is currently BC break because `DateTimeInterface` exists only since PHP 5.5.

https://secure.php.net/manual/fr/class.datetimeinterface.php

Reply:
Not anymore, since < php 5.6 is dropped.